lib/companies_web/helpers/user_helpers.ex

```diff
@@ -8,7 +8,11 @@ defmodule CompaniesWeb.UserHelpers do
     end
   end
 
-  def current_user(conn), do: conn.assigns[:current_user]
+  def current_user(%{assigns: %Phoenix.LiveView.Socket.AssignsNotInSocket{}}), do: false
```

Comment:
This should return `nil`, otherwise, [signed_in/1](https://github.com/beam-community/elixir-companies/blob/538c03d081d3cdc3981689e7b5710ff384acb739/lib/companies_web/helpers/user_helpers.ex#L17) function will return true even if the user is not logged in